CGPs/CGP-0043.md

```diff
@@ -0,0 +1,60 @@
+---
+cgp: 0043P
+title: Reserve freezing to get back to target allocation
+date-created: 2021-10-12
+author: @martinvol, @rcroessmann
+status: DRAFT
+governance-proposal-id: [if submitted]
```

Comment:
this needs a `discussions-to` forum post link.

Reply:
```suggestion
discussions-to :https://forum.celo.org/t/economic-parameters-bucket-sizes-and-reserve-allocation-cgp-40-cgp-43/1997
governance-proposal-id: [if submitted]
```